app/src/main/res/xml/feed_settings.xml

```diff
@@ -57,6 +57,14 @@
         android:summary="@string/feed_volume_reduction_summary"
         android:title="@string/feed_volume_reduction" />
 
+    <ListPreference
```

Comment:
Please use `MaterialListPreference` (might be only available if you merge `develop`), so that the style matches Material Design 3

Reply:
Done! Thanks for catching that.